tests/acceptance/features/coreApiShareManagementToShares/moveReceivedShare.feature

```diff
@@ -21,7 +21,7 @@ Feature: sharing
     And user "Brian" moves folder "/Shares/TMP" to "/Shares/new" using the WebDAV API
     And the administrator deletes user "Carol" using the provisioning API
```

Comment:
```suggestion
    And the administrator deletes user "Carol" using the provisioning API
```
Why is this step needed and why aren't we refactoring the step from provisioning to graph?

Reply:
internally should use graph. but step cleanup can be in upcoming PRs. But it's hard to come with great naming.

Reply:
@saw-jan I might be wrong. But can't we only use the `When` step for the move (rename) step, and sharing and accepting share can be done using the `Given` step? IMHO it's better if we can do small fixes in the same PR where we encounter the problem.
Also, why are we deleting the user `Carol`?

Reply:
> why are we deleting the user Carol?

I really have no idea. what the scenario suggests?

>MHO it's better if we can do small fixes in the same PR where we encounter the problem.

Yeah, right. this type of PRs are the opportunity to fix the test if required but I don't have any guess about how much the file changes would be if we do RELATED minor refactor. Depends on @S-Panta

Reply:
I suggest that we do these things in another PR. Some scenario seems to be refactored and added in this file. There are already more changes in this PR,

Reply:
Then please, follow up with it after this is merged